### ISSUE TITLE
PHP Fatal error: Class 'parseRestClient'

### ISSUE THREAD

**Issue by atkinja** [andrewscofield/parse.com-php-library#114]
Hey team

I am getting this: PHP Fatal error: Class 'parseRestClient'

Just testing out by creating an object and here is the code:

<?php
include_once 'parseConfig.php';
include_once 'parsePHP/parseObject.php';

$parse = new parseObject('Videos');
BUT, I get the error: [03-Oct-2013 23:40:57] PHP Fatal error: Class 'parseRestClient' not found in /home3/jatkinne/public_html/twilioTest/parsePHP/parseObject.php on line 3

what is line 3? It is: class parseObject extends parseRestClient{

so what is parseRestClient? where can I get this file? or do I need to create it?

THANKS GUYS!!

**Comment by ninjaphuah** [comment]
getting the same issue.

**Comment by atkinja** [comment]
Yeah so I have skipped this route and just running curl from inside my PHP code  using CURLOPT, something like this

error_reporting(-1);
ini_set('display_errors', true);
$handle = curl_init();
$url = "https://api.parse.com/1/classes/myClass";
$headers = array (
    "Content-Type: application/json",
    "X-Parse-Application-Id: APPID",
    "X-Parse-REST-API-Key: RESTID"
);

curl_setopt($handle, CURLOPT_URL, $url);
curl_setopt($handle, CURLOPT_HTTPHEADER, $headers);
curl_setopt($handle, CURLOPT_SSL_VERIFYPEER, false);
curl_setopt($handle, CURLOPT_RETURNTRANSFER, true);
curl_setopt($handle, CURLOPT_POST, true);
curl_setopt($handle, CURLOPT_HTTPGET, true);

$response = curl_exec($handle);

print "$response";
///

**Comment by SinanEker** [comment]
The parseRestClient class is located in the parse.php file. You have to include it in your script. If you include it you don't need to include parseConfig.php because all parse library files are included in parse.php

**Comment by ninjaphuah** [comment]
It looks like the problem lies with header name and the class name don't match and it is required by some of the MVC framework.

**Comment by iliyaskhan** [comment]
I am following atkinja's method and works well but  I want to know what if I need some specific data  with particular key. Is it possible to do so? I need all records which are having a particular id in it.

**Comment by andrewscofield** [comment]
No, usually that error means there is something wrong with the server you
are on. It should give you more of an error message, can you post that?

**Comment by NandoSantana** [comment]
parseRestClient class are on file parse.php, include to on your project.
